news/2024/2024-01-13-new-featured-artist-anubasu-anubasu.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "New Featured Artist: anubasu-anubasu"
+date: 2024-01-13 19:00:00 +0000
+---
+
+Send your warm welcomes to our first Featured Artist of the year: **anubasu-anubasu**!
+
+![](https://assets.ppy.sh/artists/397/header.jpg)
+
+Fusing trance, hardcore, and hardstyle in their own one-of-a-kind way, [**anubasu-anubasu**](https://osu.ppy.sh/beatmaps/artists/395) made a name for themselves in osu! through iconic maps like [Materialize Anything](https://osu.ppy.sh/beatmapsets/1847997) and [Mercury Prayer](https://osu.ppy.sh/beatmapsets/603755). And as of today, that name is officially part of our game forever!
+
+Pick your favourite of **7** outstanding songs from [**anubasu-anubasu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/397) (or choose all 7 if your mapping urges are insatiable!).
+ 
+As you've heard a billion times by now, osu!(lazer) supports performance points now. If you haven't started playing on the new client yet, here's a taste of what you're missing out on:
```

Comment:
no not yet

Reply:
preparing news posts early was a mistake